resources/test/dumps/Bacteria Hunt Speeds.js

```diff
@@ -165,7 +168,15 @@ var procedures = (function() {
           procedures["MAKE-FLAGELLA"]();
           SelfManager.self().setXY(Prims.randomCoord(world.topology.minPxcor, world.topology.maxPxcor), Prims.randomCoord(world.topology.minPycor, world.topology.maxPycor));
         }, true);
-      }), [1, 2, 3, 4, 5, 6]);
+      }, "[ this-variation ->
```

Comment:
I think this is actually my misunderstanding. For some reason, I thought the model js gets loaded into the page in a `<script>` tag, which would introduce real risks. Of course, it doesn't happen this way (perhaps in widgets, as you mention). I was also shocked by the github highlighting and assumed it was due to lack of HTML-escaping, but that may be unrelated.